docs/user-guide/src/bmo/ironic_installation.md

```diff
@@ -0,0 +1,136 @@
+# Install Ironic
+
+Baremetal Operator must be able to interact with Ironic. First,
+we will see what are the Ironic optional containers we need to
+run. After that we will go through Ironic installation in-cluster and
+out-of-cluster. In any case, there are a couple of containers that need
+to run in order to provision baremetal nodes.
+
+- ironic
+- ironic-inspector
+- ironic-endpoint-keepalived
+- ironic-log-watch
+- ipa-downloader
+- dnsmasq
+- httpd
+
+To know more about each container's functionality check the documentation
+[here](https://github.com/metal3-io/ironic-image#description).
+
+## Prerequisites
+
+Container runtime (e.g., docker, podman). Here we use docker.
+
+## Environmental variables
+
+<!-- include these inline so we can style an match variables -->
+{{#include ironic_variables.md}}
+
+## Optional containers
+
+In some circumstances, it might be necessary to install some additional containers on
+besides the mandatory Ironic containers. And this might depend on whether your target nodes are
+virtual or physical machines, or whether you want to use more stable BMC protocol
+like Redfish and IPMI. Let's check why we would need these optional containers.
+We run these optional containers on the host machine, i.e., out-of-cluster regardless
+of the Ironic installation method.
+
+### httpd
+
+We use this container to run an Apache web server for hosting node images via HTTP or via HTTPS when TLS enabled. Below is the list of images (from `$IMAGES_PATH`) that we mount into the httpd container's `/shared` path.
+
+1. Ironic Python Images (IPA) via http for PXE booting
+    - ironic-python-agent.kernel
+    - ironic-python-agent.initramfs
+
+2. Target node image in qcow2 format
+    - bionic-server-cloudimg-amd64-disk.img
```

Comment:
you meant:
```suggestion
    - bionic-server-cloudimg-amd64-disk.qcow2
```
?

Reply:
no, either if fine for me

Reply:
since you mentioned qcow format, it makes sense to note it accordingly